README.md

```diff
@@ -1,8 +1,58 @@
+If you don't have homebrew you can get it [here](https://brew.sh/).
+2. You need [watchman](https://facebook.github.io/watchman/docs/install.html). If you don't have it, install it with `brew install watchman`. If you want to check whether or not you have it type `watchman -v`. If you have version 4.9.4 or above, you don't need to do anything. If you don't, upgrade it with `brew upgrade watchman`.
+3. You need [expo](https://expo.io/). Expo allows you to build, deploy and iterate on your React Native app very quickly. Install expo globbally using `yarn globall add expo-cli`.
+Run `expo --version` on your terminal window to ensure it installed correctly. You should see something like 1.1.0 (or the version you installed).
+4. You need to clone this repo.
+5. Finally, to install all the dependencies this project requires, go to the folder where you cloned the repo and run `yarn install`.
+
+## IDE
+We recomnend [Visual Studio Code](https://code.visualstudio.com/). It's also a widely used option at Hudl.
+
+## Running the app
+Congratulations, if you've made it here you have your development environment ready :tada:. Now, you need a place to deploy the fancy animations you'll be creating.
+
+There are 2 options: Running it on the device and on the simulator. Ideally, we could all have them running on the devices. That requires your phone and computer to be on the same network. However, since we don't know the quality of network we'll have on the venue, we highly recommend that you also install one of the simulators as a fallback option in case network isn't good enough.
+
+## Option 1: Run it on your iOS or Android simulator.
+1. Install iOS or Android simulators, if you don't have them already:
+    1. For iOS, [install Xcode](https://itunes.apple.com/es/app/xcode/id497799835?l=en&mt=12).
+    2. For Android, [install Android Studio](https://developer.android.com/studio). This is your only viable option if you're on a Windows machine.
+2. Open 2 terminal windows.
+    1. In the first one, run `yarn storybook`.
+    2. In the second terminal window, run `yarn start`. That should open a new tab in your browser where expo would start doing it's magic.
+    3. In the tab thas was just opened, click on `Run on Android device / emulator` or `Run on iOS simulator` depending on what you installed in the first step.
+<img width="288" alt="How expo looks like" src="https://user-images.githubusercontent.com/2914389/54987283-9b511280-4fb4-11e9-8cd9-164e08bb0eb4.png">
+After your simulator launches, you should see something like this.
+    ![running expo](https://user-images.githubusercontent.com/2914389/54988086-051dec00-4fb6-11e9-8c6e-6bcb58ef06ed.gif)
+
+You're all set :slightly_smiling_face:.
+
+### Option 2: Run it on your Android or iOS device
+1. Create an account on [expo.io](https://expo.io/)
```

Comment:
not sure this is required? I don't remember creating an expo account.

Reply:
How does the app on your phone know about your project then? I couldn't find any other option.

If we can remove 1 step it'd be great.

Reply:
I think the QR code is something like `exp://192.168.1.xxx`

Reply:
I know you can publish snacks to expo. In that case you do need an account.

Reply:
I checked my phone and it seems like I am logged into expo, so you might be right here.